kubernetes/monitoring/deploy-secrets.sh

```diff
@@ -105,6 +106,14 @@ elif [ "$secret" == "log-clustering-secrets" ]; then
     # cmsmonit keytab
     cmsmonit_f=`ls $sdir/cmsmonit-keytab/ | awk '{ORS=" " ; print "--from-file="D"/"$1""}' D=$sdir/cmsmonit-keytab | sed "s, $,,g"`
     files="${log_f} ${cmsmonit_f}"
+elif [ "$secret" == "monit-mongo-secrets" ]; then
+mongo_ru=`cat $sdir/monit-mongo-secrets/secrets | grep MONGO_ROOT_USERNAME | awk '{print $2}'`
```

Comment:
could you please use proper indent for code inside `elif`, it is small thing but it is easy for eyes when opened in editor.

Reply:
Thanks Valentin, i forget to rollback a temporary change. Now I made it more readable.